experimental/virtio/src/queue/mod.rs

```diff
@@ -35,8 +37,8 @@ pub struct DriverWriteOnlyQueue<const QUEUE_SIZE: usize, const BUFFER_SIZE: usiz
 impl<const QUEUE_SIZE: usize, const BUFFER_SIZE: usize>
     DriverWriteOnlyQueue<QUEUE_SIZE, BUFFER_SIZE>
 {
-    pub fn new() -> Self {
-        let inner = Queue::new(DescFlags::empty());
+    pub fn new<VP: Translator>(translate: VP) -> Self {
```

Comment:
I'm still unconvinced of this trait; I would either make it a proper trait with a `translate` method, and then call the object implementing it `translator`, or just define it as a function, in which case I would call the object `translate`. Up to you anyways.

Reply:
Yeah, you're probably right, we should have a proper translator trait with methods that go both ways. But yet again I'm going to punt it to the pile of future clean-ups :)